### PR TITLE
Pivotal ID # 184068754: Implementation 6.1 test case in e2es

### PR DESCRIPTION
**Pull request by msrada94**
https://www.pivotaltracker.com/story/show/184068754

List of the changes:
* add scenario with preferred sources and files changed

### CODE REVIEW

File: src/test/resources/features/6-fileSource.feature [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+    ReleaseDate	2021-02-12
+
+    Study	SECT-001
+    Title	Root Section
+    File List	FileList.json
+
+    File	File1.txt
+
+    """
+    And the variable "SUBMISSION" with value "SUBMISSION"
```
Comment (Juan-EBI) [inline src/test/resources/features/6-fileSource.feature:182]: we can simplify by using constants directly. Not point in declare variables only yo hold constant values.

File: src/test/resources/features/6-fileSource.feature [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+  Considers submissions and resubmissions with different source files as: user space, group, fire, bypassing files with fire.
+
+  Background:
+    Given the setup information
+      | environmentUrl | http://localhost:8080        |
+      | ftpUrl         | /Users/miguel/Biostudies/ftp |
+      | userName       | admin_user@ebi.ac.uk         |
+      | userPassword   | 123456                       |
+    * the variable "storageMode" with value "NFS"
```
Comment (Juan-EBI) [inline src/test/resources/features/6-fileSource.feature:11]: we already agree to set storage mode as part of environment

**Review by Juan-EBI — APPROVED** [review]
Please address comments before merge

File: src/test/resources/features/6-fileSource.feature (hunk 2 above)
Comment (msrada94) [reply src/test/resources/features/6-fileSource.feature:11]: yes, my fault

File: src/test/resources/features/6-fileSource.feature (hunk 1 above)
Comment (msrada94) [reply src/test/resources/features/6-fileSource.feature:182]: you're right!

**Pull request merged**